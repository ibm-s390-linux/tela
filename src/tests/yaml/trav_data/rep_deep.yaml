root:
  map1:
    map2:
      map3:
        map4:
          map5:
            map6:
              map7:
                map8:
                  replace:
                    map9:
                      map10:
