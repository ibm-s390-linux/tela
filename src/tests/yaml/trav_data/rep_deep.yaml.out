Before (root=non-null):
==================================
  root:
    map1:
      map2:
        map3:
          map4:
            map5:
              map6:
                map7:
                  map8:
                    replace:
                      map9:
                        map10:
==================================

Callback:
==================================
root                                    : root:
root/map1                               :   map1:
root/map1/map2                          :     map2:
root/map1/map2/map3                     :       map3:
root/map1/map2/map3/map4                :         map4:
root/map1/map2/map3/map4/map5           :           map5:
root/map1/map2/map3/map4/map5/map6      :             map6:
root/map1/map2/map3/map4/map5/map6/map7 :               map7:
root/map1/map2/map3/map4/map5/map6/map7/map8:                 map8:
root/map1/map2/map3/map4/map5/map6/map7/map8/replace:                   replace:
*** Replacing node
==================================

After (root=non-null):
==================================
  root:
    map1:
      map2:
        map3:
          map4:
            map5:
              map6:
                map7:
                  map8: replacement
==================================
